src/test/java/fi/otavanopisto/kuntaapi/test/PtvServiceChannelMocker.java

```diff
@@ -0,0 +1,84 @@
+package fi.otavanopisto.kuntaapi.test;
+
+import static org.junit.Assert.fail;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Map;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.fasterxml.jackson.core.type.TypeReference;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import fi.metatavu.ptv.client.model.V4VmOpenApiElectronicChannel;
+import fi.metatavu.ptv.client.model.V4VmOpenApiPhoneChannel;
+import fi.metatavu.ptv.client.model.V4VmOpenApiPrintableFormChannel;
+import fi.metatavu.ptv.client.model.V4VmOpenApiServiceLocationChannel;
+import fi.metatavu.ptv.client.model.V4VmOpenApiWebPageChannel;
+
+public class PtvServiceChannelMocker extends AbstractPtvMocker<Object> {
+
+  private static final String FAILED_TO_READ_MOCK_FILE = "Failed to read mock file";
+  private static final String BASE_PATH = "/ptv/api/v4/ServiceChannel";
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Refactor your code to get this URI from a customizable parameter. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1075)